authbridge/authlib/plugins/toolprune/plugin.go

```
@@ -0,0 +1,554 @@
+	if fallback.set() {
+		return fallback, rateConfigured
+	}
+	if r, ok := defaultPricing[key]; ok {
+		return r, rateDefault
+	}
+	return modelRates{}, rateNone
+}
+
+// configuredPricing reports whether the operator supplied any rates of their
+// own, as opposed to relying on the built-in defaults.
+func (c *config) configuredPricing() bool {
```

golangci-lint (2.13.2)

[error] func (*config).configuredPricing is unused

(unused)